zerver/webhooks/codebase/doc.md

```diff
@@ -0,0 +1,16 @@
+Get Zulip notifications for your for Codebase events!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to your Codebase Dashboard, and click on the settings icon in
+   the top-right corner. Click on **Event Hooks**.
+   Now, click **New Event Webhook**.
```

Comment:
I would join the last two sentences together, `Click on **Event Hooks**, and click **New Event Webhook**.` It reads better, imo!